### DISCUSSION TITLE
new true and false validation rules

### DISCUSSION THREAD

**Discussion by y8a5y** [laravel/framework#50371] [Ideas] [answered]
I thought it would be a somewhat good idea to add `true` and `false` validation rules.

Sometime we don't really care about the boolean value in itself and only need to check it's value once, e.g. for a EULA checkbox.

With those rules we wouldn't have to check after the request with something `if ($request->boolean("eula")) {...}` and would instead only need `"eula" => ["true", "required"]` in the validation rules.

If others think it's a good idea then I may have the knowledge to implement it myself, tho a bit crudely, so any insight is welcome.

**Comment by smares** [comment, marked as answer]
https://laravel.com/docs/10.x/validation#rule-accepted and https://laravel.com/docs/10.x/validation#rule-declined

**Reply by y8a5y** [reply]
hoooo yes thanks, for some reason I remembered having saw something like that but couldn't find it. Should've read more thoroughly.
